synfig-studio/src/gui/app.cpp

```diff
@@ -1389,7 +1393,9 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 	ACCEL("<Control><Shift>less",    "<Actions>/canvasview/seek-begin");
 	ACCEL("<Control><Shift>greater", "<Actions>/canvasview/seek-end");
 	ACCEL("<Mod1>o",                 "<Actions>/canvasview/toggle-onion-skin");
-	ACCEL("0",                       "<Actions>/canvasview/canvas-zoom-fit");
+	ACCEL("<Control>equal",									"<Actions>/canvasview/canvas-zoom-in"					);
+	ACCEL("<Control>minus",									"<Actions>/canvasview/canvas-zoom-out"					);	
+	ACCEL("<Control>0",                       "<Actions>/canvasview/canvas-zoom-fit");
```

Comment:
Shortcuts don't work because those paths are already set with keys above `equal minus 0`

You have to change the name to something like this:
`"<Actions>/canvasview/canvas-zoom-in-2"`

Also, add those actions in `canvasview.cpp` file

Reply:
Got it , I will make the required changes. Thank you

Reply:
Resolved :)